mon-entreprise/source/pages/Gérer/Home.tsx

```diff
@@ -276,6 +276,22 @@ export default function Gérer() {
 								</p>
 							</Trans>
 						</Link>
+						<a
+							className="ui__ interactive card box lighter-bg"
+							href="https://www.monidenum.fr"
+							target="_blank"
+							rel="noreferrer noopener"
+						>
+							<Trans i18nKey="gérer.ressources.kbis">
+								<h3 className="ui__ h h5">Récupérer un extrait de Kbis</h3>
+								<p className="ui__ notice">
+									Le Kbis est un document permettant de justifier de
+									l'enregistrement de l'entreprise au RCS et de prouver son
```

Comment:
J'expliciterai “RCS”, soit avec la balise `abbr`

```suggestion
									l'enregistrement de l'entreprise au <abbr title="Registre du Commerce et des Sociétés">RCS</abbr> et de prouver son
```

soit en remplaçant directement par “registre des sociétés” dans le texte